packages/skills/carpark_detection/behaviours.py

```diff
@@ -125,7 +125,8 @@ def __init__(self, **kwargs):
         """Initialise the behaviour."""
         self._services_interval = kwargs.pop('services_interval', 30)  # type: int
         super().__init__(**kwargs)
-        self._last_connection_status = self.connection_status.is_connected
+        self._last_connection_status = self.context.connection_status.is_connected
+        print("__init__ self._last_connection_status {}".format(self._last_connection_status))
```

Comment:
Please use `logger.info` rather than print.

Reply:
oops that was left in by accident I'll remove it.